cmd/thanos/tools.go

```diff
@@ -36,29 +38,41 @@ func registerCheckRules(app extkingpin.AppClause) {
 
 func checkRulesFiles(logger log.Logger, files *[]string) error {
 	var failed errutil.MultiError
+	errhandle := func(e error) {
+		level.Error(logger).Log("result", "FAILED", "error", e)
+		level.Info(logger).Log()
+		failed.Add(e)
+	}
 
 	for _, fn := range *files {
 		level.Info(logger).Log("msg", "checking", "filename", fn)
```

Comment:
This isn't a filename anymore but rather a pattern:
```suggestion
		level.Info(logger).Log("msg", "checking", "pattern", fn)
```